docs/text/rolling.rst

```diff
@@ -50,9 +51,11 @@ but rolling works on all 3 data formats :ref:`data-formats-label`):
 | 2  | t9   | 11 | 13 |
 +----+------+----+----+
 
-where you have measured two values (x and y) for two different entities (1 and 2) in 4 or 2 time steps.
+where you have measured the values from two sensors x and y for two different entities (id 1 and 2) in 4 or 2 time
+steps (t1 to t9).
 
-If you set `rolling` to 0, the feature extraction works on
+Now, we can use :func:`tsfresh.utilities.dataframe_functions.roll_time_series` to get consecutive sub-time series.
+E.g. if you set `rolling` to 0, the feature extraction works on
```

Comment:
Rolling = 0 is not possible with this function. It is just using the function not at all.